server/config/subject_page.proto

```diff
@@ -261,6 +261,40 @@ message MapTileSpec {
   string geo_json_prop = 3;
 }
 
+// Spec for getting value from property values api call
+message PropertySpec {
+  string property = 1;
+  string direction = 2;
```

Comment:
describe the fields.  should direction be an enum?

Reply:
done